src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/model/courtcaseservice/CCSMatchIdentifiers.java

```diff
@@ -18,12 +21,20 @@ public class CCSMatchIdentifiers {
     private final String crn;
     private final String pnc;
     private final String cro;
+    private final List<CCSOffenderAlias> aliases;
 
     public static CCSMatchIdentifiers of(MatchIdentifiers matchIdentifiers) {
         return CCSMatchIdentifiers.builder()
                 .crn(matchIdentifiers.getCrn())
                 .pnc(matchIdentifiers.getPnc())
                 .cro(matchIdentifiers.getCro())
+                .aliases(mapAliases(matchIdentifiers))
                 .build();
     }
+
+    private static List<CCSOffenderAlias> mapAliases(MatchIdentifiers matchIdentifiers) {
+        return Optional.ofNullable(matchIdentifiers.getAliases()).map(offenderAliases ->
+                        offenderAliases.stream().map(CCSOffenderAlias::of).collect(Collectors.toList()))
+                .orElse(null);
```

Comment:
Could this be returns an Optional.empty() instead?

Reply:
It is the usual way but doesn't apply to the very reason I created this utility method.. the calling code isn't expecting or wanting an Optional